Makefile

```diff
@@ -82,7 +82,15 @@ test-coverage:
 # compile for multiple platforms
 .PHONY: cross
 cross:
-	gox -osarch="darwin/amd64 linux/amd64 windows/amd64" -output="dist/bin/{{.OS}}-{{.Arch}}/odo" $(BUILD_FLAGS) ./cmd/odo/
+	@for platform in linux darwin windows ; do \
+		if [ $$platform == "windows" ]; then \
+			echo "Cross compiling $$platform-amd64 and placing binary at dist/bin/$$platform-amd64/"; \
```

Comment:
You can put the echo before if as its common

Reply:
Could not get it. Can you please explain it

Reply:
Got it, thanks
Will update it

Reply:
Done